cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -507,7 +507,7 @@ rules:
 ---
 # The role for the operator to manage resources in its own namespace
 apiVersion: rbac.authorization.k8s.io/v1beta1
-kind: Role
+kind: ClusterRole
```

Comment:
We can't change this to a `ClusterRole`. Are the MDBs actually namespaced? If so, this could remain a `Role`. But if the mdbs are not namespaced, please add the new mdb rules next to the pdb rules in the `rook-ceph-global-rules` ClusterRole.

Reply:
makes sense! done.